File: charcutaria-business-manager/src/app/shared/models/jwtUserInfo.ts
export interface JWTUserInfo {
    userId: number;
    corpClientId: number | null;
    username: string;
    name: string;
    currency: string;
    active: boolean;
    roleId: Number;
    createdOn: Date;
    lastUpdated: Date | null;
    dBAName: string;
    companyName: string;
}

File: Charcutarie.Api/Charcutarie.Models/ViewModels/JWTUserInfo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Charcutarie.Models.ViewModels
{
    public class JWTUserInfo
    {
        public long UserId { get; set; }
        public int? CorpClientId { get; set; }
        public string Username { get; set; }
        public string Name { get; set; }
        public string Role { get; set; }
        public string Currency { get; set; }
        public bool Active { get; set; }
        public int RoleId { get; set; }
        public DateTimeOffset CreatedOn { get; set; }
        public DateTimeOffset? LastUpdated { get; set; }
        public string DBAName { get; set; }
        public string CompanyName { get; set; }
    }
}
